### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by jderusse**
```
1.12.0 Core
1.1.0  CloudFormation
0.1.3  CloudFront
-      CloudWatch
1.2.0  CloudWatchLogs
1.2.1  CodeDeploy
1.3.0  CognitoIdentityProvider
1.1.1  DynamoDb
0.1.3  Ecr
-      ElastiCache
1.2.0  EventBridge
-      Firehose
1.1.1  Iam
0.1.1  Kinesis
1.5.0  Lambda
0.1.3  RdsDataService
0.1.5  Rekognition
0.1.1  Route53
1.9.2  S3
0.1.1  SecretsManager
1.4.1  Ses
1.2.1  Sns
1.6.0  Sqs
1.2.1  Ssm
0.1.1  StepFunctions


1.0.1  dynamo-db-session
1.0.1  flysystem-s3
0.2.2  illuminate-cache
0.1.4  illuminate-mail
```

### CODE REVIEW

File: src/Integration/Flysystem/S3/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## NOT RELEASED
 
+## 1.0.1
+
+### Fixed
+
+- Fixed bucket used in `copy` calls
```
Comment (Nyholm) [inline src/Integration/Flysystem/S3/CHANGELOG.md:9]: Removed `urlencode()`  and added bucket to `CopySource` in the `copy()` method. This comply with the Flysystem interface and the AWS adapter.

File: src/Integration/Laravel/Filesystem/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ## NOT RELEASED
 
+## 0.2.2
+
+### Fixed
+
+- Fix laravel locking to use consistent reads
```
Comment (Nyholm) [inline src/Integration/Laravel/Filesystem/CHANGELOG.md:9]: This changelog belongs to src/Integration/Laravel/Cache/CHANGELOG.md

File: src/Core/CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,12 @@
 
 ## NOT RELEASED
 
-### Added
+## 1.12.0
```
Comment (Nyholm) [inline src/Core/CHANGELOG.md:5]:
```suggestion
## 1.12.0

### Added
```

File: src/Service/CloudFront/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 ## NOT RELEASED
 
+## 0.1.3
+
+### Fixed
+
+- Fixed camelCased of Dom classes
```
Comment (Nyholm) [inline src/Service/CloudFront/CHANGELOG.md:9]: We also need this https://github.com/async-aws/aws/pull/1010

```suggestion
- Fixed camelCased of Dom classes
- Fixed invalid operation names
```

File: src/Service/CloudWatchLogs/CHANGELOG.md [hunk 5]
```diff
@@ -2,13 +2,14 @@
 
 ## NOT RELEASED
 
-### Fixed
+### 1.2.0
```
Comment (Nyholm) [inline src/Service/CloudWatchLogs/CHANGELOG.md:5]:
```suggestion
## 1.2.0

### Fixed
```

File: src/Service/Ecr/CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,8 @@
 
 ## NOT RELEASED
 
+## 0.1.3
```
Comment (Nyholm) [inline src/Service/Ecr/CHANGELOG.md:5]: Maybe it is time for 1.0? 
It has over 1000 downloads.

File: src/Service/Kinesis/CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,12 @@
 
 ## NOT RELEASED
 
+## 0.1.1
```
Comment (Nyholm) [inline src/Service/Kinesis/CHANGELOG.md:5]: Maybe 1.0?

File: src/Service/Rekognition/CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,8 @@
 
 ## NOT RELEASED
 
+## 0.1.5
```
Comment (Nyholm) [inline src/Service/Rekognition/CHANGELOG.md:5]: We should also add that we added operations `getCelebrityInfo()` and `recognizeCelebrities()`

File: src/Service/Route53/CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,8 @@
 
 ## NOT RELEASED
 
+## 0.1.1
+
 ### Added
 
 - AWS enhancement: Documentation updates for route53.
```
Comment (Nyholm) [inline src/Service/Route53/CHANGELOG.md:9]: Maybe remove this line

Comment (Nyholm) [reply src/Service/Route53/CHANGELOG.md:9]: We also fixed case on \DOMElement 

> Fixed camelCased of Dom classes

File: src/Service/StepFunctions/CHANGELOG.md [hunk 10]
```diff
@@ -2,6 +2,16 @@
...
+## 0.1.1
+
+## Added
+
+- Added operation `sendTaskFailure`
+- Added operation `sendTaskHeartbeat`
+- Added operation `sendTaskSuccess`
+
+First version
+
```
Comment (Nyholm) [inline src/Service/StepFunctions/CHANGELOG.md:14]:
```suggestion

```

**Review by Nyholm — CHANGES_REQUESTED** [review]
Awesome. Well done. 

I have a few changes and suggestions. Feel free to make the releases after these are corrected.

File: src/Integration/Laravel/Filesystem/CHANGELOG.md (hunk 2 above)
Comment (jderusse) [reply src/Integration/Laravel/Filesystem/CHANGELOG.md:9]: good catch !!

**Pull request merged**